README.md

```diff
@@ -62,7 +62,7 @@ package-hosted directory like this:
 
 ```bash
 cd cloud-controller-manager/cmd/digitalocean-cloud-controller-manager
-REGION=fra1 DO_ACCESS_TOKEN=your_access_token go run main.go \
+REGION=fra1 DO_ACCESS_TOKEN=your_access_token DO_IP_ADDR_FAMILIES=ipv4 go run main.go \
```

Comment:
I'd omit the usage from this example since it's not strictly needed when running locally.

Reply:
Fixed